CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `asb` resumes unfinished swaps upon startup.
   This is a breaking changed because we now save the peer-id of the counterparty in the database.
   It is recommended to reset the database before running the upgraded version of the `asb` because any unfinished swaps will be tried for resume but will fail without the counterparty peer-id being present.
+- The `swap` CLI saves the seller-peer-id into the database to be re-used for `resume`, so the `--seller-peer-id` parameter for resume was removed.
```

Comment:
I know it is not always easy to do but I think it would be worth it to strive for a wording where these entries use the "heading" as their first word.

Like:

**Added** _a changelog file_.

Additionally, I think we should word it in a "user-first" mentality. What is important to the user is that the `resume` command **changed**, not that we added stuff to the database.

My suggestion would be to move this to the **Changed** section and say:

### Changed

- The `resume` command of the `swap` CLI to no longer require the `--seller-peer-id` parameter. This information is now saved in the database.